src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -141,6 +142,7 @@ export class StepIndexPattern extends Component<StepIndexPatternProps, StepIndex
       initialQuery || context.services.uiSettings.get(UI_SETTINGS.INDEXPATTERN_PLACEHOLDER);
     this.state.indexPatternName = indexPatternCreationType.getIndexPatternName();
     this.dataSrouceEnabled = context.services.dataSourceEnabled;
+    this.fetchIndices = debounce(this.fetchIndices.bind(this), 700) as any;
```

Comment:
Out of curiosity, how heavy of a load are we talking? I like the debounce here, but i wonder if we can take the delay lower. Also instead of reassigning `fetchIndices` to itself, can you assign it to a new variable like `debouncedFetchIndices`

Reply:
+1, I can't see where else this is used by I don't see it be explicitly private.